contracts/WROSE.sol

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.19;
+
+/*
+ * This WROSE allows meta-withdrawals to accounts that do not have gas.
+ * Token transfers are not private because Sapphire compute node operators can see which accounts' storage slots are active.
+ * Do NOT use this code for anything that requires privacy if you do not trust Sapphire compute node operators.
+ */
+
+import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+
+contract WROSE {
+    using ECDSA for bytes32;
+
+    mapping(address => uint256) private balances;
+    mapping(uint256 => bool) public replayNonce;
+
+    function balanceOf() public view returns (uint256) {
+        return balances[msg.sender];
+    }
+
+    function deposit() public payable {
```

Comment:
Any chance of a `depositTo` function to allow people to deposit into another persons account?

Reply:
The idea is 
`deposit`   moves ROSE from wallet to contract
`withdraw` moves ROSE from contract to original wallet
`metaWithdraw` moves ROSE from contract to any wallet.

TLDR: metaWithdraw is the same as `depositTo`